examples/scroll/lv_example_scroll_1.c

```diff
@@ -1,15 +1,56 @@
 #include "../lv_examples.h"
 #if LV_BUILD_EXAMPLES
+#include <stdio.h>  /* For sprintf() prototype. */
```

Comment:
> We have `lv_snprintf` 😎

Outstanding!  Actually, I just realized with the LV_LOG_USER(), I think the `<stdio.h>` and `lv_snprintf` are no longer needed.  So I will remove that include line since it is now inapplicable.